src/item.py

```diff
@@ -0,0 +1,13 @@
+class Item():
+  def __init__(self, name, description):
+    self.name = name
+    self.description = description
+
```

Comment:
Kat , All of your classes look great , especially the return_outside in your player.py